[CBRD-23624] Regular expression functions

hgryoo opened the pull request:
http://jira.cubrid.org/browse/CBRD-23624

- Add a new tab(Regular Expressions Functions and Operators)
- Move REGEXP and RLIKE part into the new tab
- Add newly added functions
  * REGEXP_COUNT
  * REGEXP_INSTR
  * REGEXP_LIKE
  * REGEXP_SUBSTR
  * REGEXP_REPLACE
- Describe ECMAScript regex grammar
- Add serveral examples and notes

hgryoo committed:
add templates for regexp functions

hgryoo committed:
add draft

hgryoo committed:
add section of regular expressions functions

hgryoo committed:
add examples and notes

shparkcubrid commented on en/sql/function/regex_fn.rst:
```diff
@@ -0,0 +1,727 @@
+    regexp_replace('12345abcdeabcde', '[a-d]', null)
+    ======================
+      NULL
+     
```
How does it work with empty strings?

shparkcubrid commented on en/sql/function/regex_fn.rst:
```diff
@@ -0,0 +1,727 @@
+.. note::
+
+  **Compatibility Considerations**
+
```
How about writing at the top?
It would be nice if it was written in the form of 'NOTE' in the link below.
https://www.cubrid.org/manual/en/10.2/sql/tuning.html#updating-statistics

hgryoo replied:
good point, thank you. I moved it at the top of the section.

hgryoo committed:
add some examples and typo

hgryoo committed:
move compatibility considerations at the beginning of the section

hgryoo committed:
remove blank line warnings

hgryoo committed:
revise manual

- remove character equivalent section (collating element does not support)
- move ecmascript syntax section at the beginning of the page
- add examples for functions when pattern is empty string
- revise compatibliity considertions and move at the top of section
- add multibyte chracter comparision consideration note
- add example of digit range syntax"
- add link of ECMAScript regexp grammar standard

hgryoo committed:
add escape for backslash

hgryoo committed:
minor fix

shparkcubrid commented on en/csql.access:
```diff
@@ -0,0 +1,12 @@
+
+Time: 03/19/20 01:40:56.072 - NOTIFICATION *** file /home/hgryoo/hgryoo_cubrid/src/transaction/boot_sr.c, line 3130  CODE = -972, Tran = -1, EID = 4
+Program 'csql' (pid 19798) was connected from the host 'rnd06.cubrid.com'. (transaction index 1)
```
delete it

shparkcubrid commented on en/csql.err:
```diff
@@ -0,0 +1,135 @@
+
+Time: 03/19/20 01:40:56.070 - NOTIFICATION *** file /home/hgryoo/hgryoo_cubrid/src/query/vacuum.c, line 987  CODE = -1190, Tran = -1, EID = 1
+Stand-alone vacuum execution is started.
```
delete it

hgryoo committed:
remove useless files
